lib/wanda_web/controllers/health_controller.ex

```diff
@@ -0,0 +1,47 @@
+defmodule WandaWeb.HealthController do
+  @moduledoc """
+    Health-related functions
+  """
+
+  use WandaWeb, :controller
+  use OpenApiSpex.ControllerSpecs
+
+  alias Ecto.Adapters.SQL
```

Comment:
Instead of aliasing this, did you try `import Ecto.Query` ?

Reply:
I don't see a way to construct the query `SELECT 1` using the `Ecto.Query` DSL. It seems that you need to provide a scheme to `SELECT` from.